doteki/plugins/figlet.py

```diff
@@ -0,0 +1,39 @@
+import logging
+import pyfiglet  # type: ignore
+from typing import Any
+
+
+def run(settings: dict[str, Any]) -> str | None:
+
+    if not validate_settings(settings):
+        return None
+    text = settings.get("ascii_text")
+    font = settings.get("font", "standard")
+    result = pyfiglet.figlet_format(text, font)
```

Comment:
What if a user wants to render a number? Perhaps we can do:

```py
text = str(settings.get("ascii_text"))
```

Otherwise, it will error out with `text = 42`:

```
[figlet] ERROR: An error occurred in plugin 'figlet': 'int' object is not iterable
```

I've added a failing test (`test_int_text`) showing the expected behaviour.

Reply:
I added a line that converts the "ascii_text" parameter into a string. It works fine now. What do you think?

Reply:
Looks good to me! I'll edit it to do it all in one line, though.